### PR TITLE
add new social icons:facebook-messenger and whatsapp

### PR DESCRIPTION
**Pull request by abhishekkumar08**
Two new social icons added to SocialIcons component: facebook-messenger and WhatsApp.

### CODE REVIEW

**Comment by abhishekkumar08** [comment]
I haven't tested the changes can you tell me @flyingcircle, how should I test these locally?

**Comment by flyingcircle** [comment]
I would test them using the `react-native-elements-app`. It's one of the other repos in our github.

**Comment by abhishekkumar08** [comment]
@flyingcircle can you please tell me the steps how to do that. Thanks for the reply

**Comment by flyingcircle** [comment]
@abhishekkumar08 there's no 'facebook-messenger' icon in font-awesome (you can view the icons available here: https://oblador.github.io/react-native-vector-icons/) You'd have to change the icon type to something like 'material-community' in order to see it.

**Review by flyingcircle — CHANGES_REQUESTED** [review]
Documentation needs updated

**Comment by abhishekkumar08** [comment]
Upon checking the code ```#available-icon-sets``` links doesn't work correctly, updated in the commit  ```45732f2```, I haven't added all the types as I don't have confirmation for all the types. Can you please tell me the other types which I have not added? @flyingcircle

File: website/versioned_docs/version-3.3.1/props/icon.md [hunk 1]
```diff
@@ -194,6 +194,16 @@ type of icon set. [Supported sets here](#available-icon-sets).
 
 ---
 
+### `available-icon-sets`
```
Comment (flyingcircle) [inline website/versioned_docs/version-3.3.1/props/icon.md:197]: I don't know why you added this.

File: website/versioned_docs/version-3.3.1/props/social_icons.md [hunk 2]
```diff
@@ -199,9 +199,19 @@ title if made into a button (optional)
...
+|                                                                                                                                                                           Type                                                                                                                                                                           | Default |
+| :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :-----: |
+| social media type (angellist, codepen, envelope, etsy, facebook, facebook-messenger, flickr, foursquare, github-alt, github, gitlab, instagram, linkedin, medium, pinterest, quora, reddit-alien, soundcloud, stack-overflow, steam, stumbleupon, tumblr, twitch, twitter, google, google-plus-official, vimeo, vk, weibo, whatsapp, wordpress, youtube) |  none   |
```
Comment (flyingcircle) [inline website/versioned_docs/version-3.3.1/props/social_icons.md:204]: The only thing I meant was that the 2 items should be added to this list. But not in the `versioned_docs` folder, but in the `docs` folder.

File: website/versioned_docs/version-3.3.1/props/social_icons.md [hunk 3]
```diff
@@ -199,9 +199,19 @@ title if made into a button (optional)
...
+
+---
+
+### `available-icon-sets`
```
Comment (flyingcircle) [inline website/versioned_docs/version-3.3.1/props/social_icons.md:208]: this should not be here also.

**Review by flyingcircle — CHANGES_REQUESTED** [review]
I think you got a little confused, just add the new items to the social media type list and remove the other items that you added. Also you shouldn't change the `versioned_docs` but the `docs` files.

**Comment by abhishekkumar08** [comment]
> 
> 
> I think you got a little confused, just add the new items to the social media type list and remove the other items that you added. Also you shouldn't change the `versioned_docs` but the `docs` files.

Sorry for that, I went in flow and checked all the links. Edited the code as requested please have a look now. Thanks for the review.

**Pull request merged**